src/main/java/com/vinylteam/vinyl/service/CaptchaService.java

```diff
@@ -0,0 +1,7 @@
+package com.vinylteam.vinyl.service;
+
+public interface CaptchaService {
+    byte[] getCaptcha(String captchaId);
+
+    boolean validateCaptcha(String captchaId, String response);
```

Comment:
Please add empty row after interface/class declaration and before closing. It is our code style